packages/flutter_test/lib/src/binding.dart

```diff
@@ -1082,12 +1082,7 @@ class _LiveTestRenderView extends RenderView {
   }
 }
 
-class _EmptyStack implements StackTrace {
-  const _EmptyStack._();
-  static const _EmptyStack instance = const _EmptyStack._();
-  @override
-  String toString() => '';
-}
+final stack_trace.Chain _emptyStackTrace = new stack_trace.Chain(const <stack_trace.Trace>[]);
```

Comment:
the type here should probably be StackTrace (makes no real difference but makes our internal intent clearer)

Reply:
Done.